tests/unit/template_stargate_test.go

```diff
@@ -0,0 +1,186 @@
+package unit_test
+
+import (
+	"github.com/gruntwork-io/terratest/modules/helm"
+	. "github.com/k8ssandra/k8ssandra/tests/unit/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appsV1 "k8s.io/api/apps/v1"
```

Comment:
minor nit - can you make the alias all lower case. That seems to be the common convention.